CodingPractice/OOPPractice/assignments/MaxSizeList.py

```diff
@@ -0,0 +1,13 @@
+class MaxSizeList:
+
```

Comment:
Have a look at this article which discusses using __ and _ as instance variable prefixes and apply that here:
https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-a-single-and-a-double-underscore-before-an-object-name